package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-azurestaticwebapps",
     "displayName": "Azure Static Web Apps",
     "description": "%staticWebApps.description%",
-    "version": "0.10.1-alpha.1",
+    "version": "0.10.1-alpha.2",
```

Comment:
inb4 it wants the change in `package-lock.json` too

Reply:
Don't give Alex any ideas...

Reply:
You should run `npm version prerelease` and it'll take care of everything for you and it'll commit.

https://docs.npmjs.com/cli/v8/commands/npm-version

Reply:
> Don't give Alex any ideas...

Oh, we def need to add that... 😆